src/coreclr/nativeaot/Test.CoreLib/src/System/RuntimeTypeHandle.cs

```diff
@@ -10,17 +10,17 @@ namespace System
     [StructLayout(LayoutKind.Sequential)]
     public struct RuntimeTypeHandle
     {
-        private EETypePtr _pEEType;
+        private IntPtr _pEEType;
```

Comment:
Also, it may be nice to rename `_pEEType` to `_value` so that it is in sync with the full implementation.